src/Exceptions/InvalidCommandException.php

```diff
@@ -32,5 +36,41 @@ public function render(Console $console): void
         if ($missingArguments) {
             $console->writeln("Missing arguments: {$missingArguments}");
         }
+
+        $fixedArguments = $this->promptForMissingArguments($console);
+
+        foreach ($fixedArguments as $argument) {
+            $this->bag->add($argument);
+        }
+
+        throw new MistypedCommandException($this->consoleCommand->getName());
```

Comment:
@brendt you mentioned that you'd want this to be supported in middleware - can you expand on that?

I went this way because that's also how we handled that for CommandNotFoundException

Reply:
We can do it in another PR 👍

I actually forgot I added middleware support, and just thought of it last week, which is why I mentioned it 😁 I think, in the end, it'll be better to have all edge-cases be handled by middleware; but this will probably take some refactoring. 

How about we merge this PR, and I work on a refactor after that?

Reply:
That works for me!